src/Dijkstra/AST/Maybe.agda

```diff
@@ -4,6 +4,8 @@
    Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
 -}
 
+open import LibraBFT.Base.Util
```

Comment:
Modules under `Dijkstra` should not import `LibraBFT` modules.

Whatever is being used from `Libra.Base.Util` should be moved to 
- `Haskell.*` : if it is a "haskell" thing
- `Util.Prelude` :  otherwise

Reply:
Thanks, this was just for `obm-dangerous-magic` and was temporary.  DONE.